jobs/tests/fixtures_schemas.py

```diff
@@ -59,3 +59,23 @@ def mock_AnnotationJobParams2():
         deadline=datetime.datetime.utcnow() + datetime.timedelta(days=1),
     )
     return mockAnnotationJobParams2
+
+
+
+@pytest.fixture
+def mock_Extraction_AnnotationJobParams():
```

Comment:
1. make lowercase name
2. use typings to show what is return, no need for type in the name.